docs/api-reference/attribute-manager.md

```diff
@@ -58,6 +58,10 @@ Takes a single parameter as a map of attribute descriptor objects:
   * `attribute.update` (Function) - the function to be called when
   * `attribute.instanced` (Boolean, optional) - if this is an instanced attribute
```

Comment:
`instanced` and `isIndexed`. The naming inconsistency is annoying.

Reply:
> instanced and isIndexed. The naming inconsistency is annoying.

Yes I believe the difference comes from the fact that in WebGL instancing is not a boolean attribute, it is an instance "divisor" (a number). So I may have wanted to allow for specifying the divisor rather than just a bool. But I have had a hard time finding uses for the divisor, so I could be convinced to change this.

Worse, I am no longer sure `generic` is the right way to refer to the "single valued" vertex attributes. Generic might be related to OpenGL1/OpenGL2 differences. There really doesn't seem to be a standard term for these so let's keep if for now.